workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/RenderWorkflowInTest.kt

```diff
@@ -230,7 +230,7 @@ class RenderWorkflowInTest {
         render = { _, renderState ->
           Pair(
             renderState,
-            { newState -> actionSink.send(action { state = newState }) }
+            { newState -> actionSink.send(action("") { state = newState }) }
```

Comment:
we could add a `testAction` in the testing package to avoid this kind of boilerplate/frustration?

Reply:
That's a good point. Or really, the tests in question could whip that up themselves, it's not like this is a common problem for feature developers.

Reply:
Now that I've convinced myself we shouldn't police for empty strings, inclined to leave the tests as is.